src/Symfony/Component/Console/Command/Command.php

```diff
@@ -605,6 +605,10 @@ public function getUsages()
      */
     public function getHelper($name)
     {
+        if (null === $this->helperSet) {
+            throw new LogicException(sprintf('Unable to retrieve helper "%s" because there is no HelperSet defined for this command. Did you forget to call %s::setHelperSet()?', $name, get_class($this)));
```

Comment:
@xabbuh I simplified it by removing the `Did you forget...` part, let me know if you think that something like:
> Did you forget to set an application for your command via Command::setApplication()? You can also set an HelperSet directly from your command using Command::setHelperSet().

Could be helpful. Thanks